tests/unit/sagemaker/jumpstart/curated_hub/test_curated_hub.py

```diff
@@ -0,0 +1,139 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+import pytest
+from mock import Mock
+from sagemaker.jumpstart.curated_hub.curated_hub import CuratedHub
+
+REGION = "us-east-1"
+ACCOUNT_ID = "123456789123"
+HUB_NAME = "mock-hub-name"
+
+
+@pytest.fixture()
+def sagemaker_session():
+    boto_mock = Mock(name="boto_session")
+    sagemaker_session_mock = Mock(
+        name="sagemaker_session", boto_session=boto_mock, boto_region_name=REGION
+    )
+    sagemaker_session_mock._client_config.user_agent = (
+        "Boto3/1.9.69 Python/3.6.5 Linux/4.14.77-70.82.amzn1.x86_64 Botocore/1.12.69 Resource"
+    )
+    sagemaker_session_mock.account_id.return_value = ACCOUNT_ID
+    return sagemaker_session_mock
+
+
+def test_instantiates(sagemaker_session):
+    hub = CuratedHub(hub_name=HUB_NAME, region=REGION, sagemaker_session=sagemaker_session)
+    assert hub.hub_name == HUB_NAME
+    assert hub.region == "us-east-1"
+    assert hub._sagemaker_session == sagemaker_session
+
+
+def test_instantiates_handles_conflicting_regions(sagemaker_session):
+    conflicting_region = "us-east-2"
+
+    with pytest.raises(ValueError):
+        CuratedHub(
+            hub_name=HUB_NAME, region=conflicting_region, sagemaker_session=sagemaker_session
+        )
+
+
+@pytest.mark.parametrize(
```

Comment:
Didn't know about `mark.parametrize`- it's great, I'll include it in my future PRs